<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memberWidget</class>
 <widget class="QWidget" name="memberWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Member</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Member</string>
   </property>
  </widget>
  <widget class="QWidget" name="member_widget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>1091</width>
     <height>701</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
